torch/distributed/tensor/_ops/_pointwise_ops.py

```diff
@@ -398,6 +398,7 @@
     aten.trunc_.default,
     aten.where.self,
     aten.where.self_out,
+    aten.view.dtype,
```

Comment:
should it be the `default_strategy` instead of the pointwise strategy? https://github.com/pytorch/pytorch/blob/main/torch/distributed/tensor/_ops/_tensor_ops.py#L41

pointwise strategy should also work but it would try handling broadcasting semantics which might not be necessary for `view.dtype`?

Reply:
oh oops, thanks for pointing this out! let me fix

Reply:
changed!